File: python/src/tests/t_checksum.py
#!/usr/bin/python3
#
# $Id:$
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

r = atrshmlog.attach()

id = atrshmlog.get_checksum()

print('checksum  : ' + str(id) + ' : ')

old = atrshmlog.set_checksum(1)

print('checksum  : ' + str(old) + ' : ')

id = atrshmlog.get_checksum()

print('checksum  : ' + str(id) + ' : ')

old = atrshmlog.set_checksum(0)

print('checksum  : ' + str(old) + ' : ')

id = atrshmlog.get_checksum()

print('checksum  : ' + str(id) + ' : ')


print (' ')

exit(0);

# end of test

File: perl/src/tests/t_checksum.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$id = Atrshmlog::get_checksum();

print "checksum  : $id : \n";

$old = Atrshmlog::set_checksum(1);

print "checksum  : $old : \n";

$id = Atrshmlog::get_checksum();

print "checksum  : $id : \n";

$old = Atrshmlog::set_checksum(0);

print "checksum  : $old : \n";

$id = Atrshmlog::get_checksum();

print "checksum  : $id : \n";

print " \n";

exit (0);

# end of main
